### PR TITLE
Update Development version to 4.6.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by alameluchidambaram**
Update Development version on github to "4.6.0-SNAPSHOT".

### CODE REVIEW

File: Product/DBScripts/pom.xml [hunk 1]
```diff
@@ -1,11 +1,9 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (christophermay07) [inline Product/DBScripts/pom.xml:1]: Why did the spacing change on every file?

Comment (alameluchidambaram) [inline Product/DBScripts/pom.xml:1]: I have used maven Release Plugin to update the pom versions and that modified spacing in pom files.

**Comment by christophermay07** [comment]
retest this please

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge without further comment.

**Comment by sailajaa** [comment]
I reviewed PR and good with merge.

**Pull request merged**